### ISSUE TITLE
RFC: Allow to configure effector-logger via babel-plugin settings

### ISSUE THREAD

**Issue by AlexandrHoroshih** [effector/logger#66]
## Problem
Currently, if `effector-logger` is added via `babel-plugin`, like this:
```json
{
  "env": {
    "development": {
      "plugins": ["effector-logger/babel-plugin"]
    }
  }
}
```
There is no option to configure it in any way.

For e.g. i would prefer to keep easy one-line installation via plugin, but to disable logs to console in favor of inspector or redux-devtools

## Propolsal
Allow configure `effector-logger` via plugin options, like it works now in `attachLogger` options
```json
{
   "env":{
      "development":{
         "plugins":[
            [
               "effector-logger/babel-plugin",
               {
                  "reduxDevtools":"disabled",
                  "inspector":"disabled",
                  "console":"disabled"
               }
            ]
         ]
      }
   }
}
```

**Comment by AlexandrHoroshih** [comment]
Also currently effector-logger/babel-plugun replaces imports like `effector -> effector-logger`

It may be useful to add option to change this behaviour

**Comment by VladPobidash** [comment]
I would like to log changes only with ReduxDevtools, maybe you find any solution for disabling the global console logging?

**Comment by AlexandrHoroshih** [comment]
Fixed by recent 0.14.0 release
